metadata/metadata_service/proxy/gremlin_proxy.py

```diff
@@ -1773,6 +1773,12 @@ def get_resource_generation_code(self, *,
                                      uri: str,
                                      resource_type: ResourceType) -> GenerationCode:
         pass
+    
+    def get_popular_resources(self, *,
+                              num_entries: int,
+                              resource_types: List[str],
+                              user_id: Optional[str] = None) -> Dict[str, List]:
+        pass
```

Comment:
Can you please make sure to raise the `NotImplementedError` instead? 

ref: https://github.com/amundsen-io/amundsen/pull/1289/files

Reply:
@verdan Done.